CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 -   Now shows connected node status in side bar.
 -   It is now possible to remove a failed identity.
 -   Added reference (sessionId) on failed identities, for support.
+-   Application will now update automatically (Windows, MacOS, and AppImage for Linux)
```

Comment:
Application will automatically search for updates and prompt the user to download and install them (Windows, macOS and AppImage for Linux).

Reply:
I agree